src/options.js

```diff
@@ -0,0 +1,143 @@
+/* global $ */
+/* global chrome */
+var themeSelect = document.getElementById("theme");
+var loginForm = document.getElementById("login");
+var logoutButton = document.getElementById("logout");
+var loginButton = document.getElementById("loginbutton");
+var noLoggedInBlock = document.getElementById("nologgedin");
+var loggedInBlock = document.getElementById("loggedin");
+var accessToken = "";
+var userEmail = "";
+var time = "";
+var BASE_URL = "https://api.susi.ai";
+//var messagesHistory=[];
+function showLoggedInBlock(show){
```

Comment:
Is there any need of this block @stealthanthrax

Reply:
@ms10398 , I can't understand which block you are referring to, if you are referring to the comment on line 13 , it was a mistake and ill remove it

Reply:
showLoggedin and password change as this functions are not there so please remove it as it has no use for the login feature!!